src/common/options/global.yaml.in

```diff
@@ -2308,6 +2308,14 @@ options:
   level: advanced
   default: 10
   with_legacy: true
+- name: mon_client_hunt_on_resent
```

Comment:
resend instead of resent?

My concern with adding this new option but not enabling the new code by default is that it doesn't fix the issue, and certainly won't stop the rare teuthology test failures where this timing window is hit unless you change the config there.

I think the risk of enabling the new behavior is that a client might end up retrying a command to an unhealthy monitor forever rather than switching to another monitor. This risk could be eliminated if the new code retried the previous monitor a few times (to deal with temporary connectivity problems such as what happened in the issue), but then gave up and searched for a new monitor. With such a change I think we could confidently enable the new behavior by default.

Reply:
I'm going to enable it for that test specifically! thanks!